.version

```diff
@@ -1 +1 @@
-v1.5
```

Comment:
You don't need to dump this, the release script handles the patch level for you.